[2024-09-08T18:48:41.699+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-08T18:48:41.744+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: catchup_dag_v2.task1 backfill__2024-09-04T00:00:00+00:00 [queued]>
[2024-09-08T18:48:41.772+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: catchup_dag_v2.task1 backfill__2024-09-04T00:00:00+00:00 [queued]>
[2024-09-08T18:48:41.774+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 4
[2024-09-08T18:48:41.802+0000] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): task1> on 2024-09-04 00:00:00+00:00
[2024-09-08T18:48:41.843+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=2416) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-08T18:48:41.848+0000] {standard_task_runner.py:72} INFO - Started process 2420 to run task
[2024-09-08T18:48:41.836+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'catchup_dag_v2', 'task1', 'backfill__2024-09-04T00:00:00+00:00', '--job-id', '257', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_catchup_and_backfill.py', '--cfg-path', '/tmp/tmp3la11rxy']
[2024-09-08T18:48:41.872+0000] {standard_task_runner.py:105} INFO - Job 257: Subtask task1
[2024-09-08T18:48:42.026+0000] {task_command.py:467} INFO - Running <TaskInstance: catchup_dag_v2.task1 backfill__2024-09-04T00:00:00+00:00 [running]> on host 2abf3f0b0579
[2024-09-08T18:48:42.854+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Abhay' AIRFLOW_CTX_DAG_ID='catchup_dag_v2' AIRFLOW_CTX_TASK_ID='task1' AIRFLOW_CTX_EXECUTION_DATE='2024-09-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='backfill__2024-09-04T00:00:00+00:00'
[2024-09-08T18:48:42.860+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-08T18:48:42.869+0000] {logging_mixin.py:190} INFO - hello
[2024-09-08T18:48:42.883+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-08T18:48:42.922+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-08T18:48:42.927+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=catchup_dag_v2, task_id=task1, run_id=backfill__2024-09-04T00:00:00+00:00, execution_date=20240904T000000, start_date=20240908T184841, end_date=20240908T184842
[2024-09-08T18:48:43.096+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-09-08T18:48:43.229+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-08T18:48:43.253+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
